examples/tdigest.py

```diff
@@ -0,0 +1,51 @@
+# Copyright Contributors to the Cellarium project.
+# SPDX-License-Identifier: BSD-3-Clause
+
+"""
+Example: One-pass calculation of feature median using t-digest statistics
+=========================================================================
+
+This example shows how to calculate non-zero median of log normalized feature count
```

Comment:
I believe you're not log transforming, right?

Reply:
Good catch! Fixed it.